src/cloud-api-adaptor/test/e2e/README.md

```diff
@@ -131,6 +131,7 @@ Use the properties on the table below for AWS:
 |aws_vpc_sg_id|AWS VPC Security Groups ID||
```

Comment:
It would be good to mention that e2e tests uses the tag `caa-e2e-test`. Also requirements for OIDC setup.

Reply:
I will send it in a part-3 PR, ok?